app/address/[address]/layout.tsx

```diff
@@ -31,6 +31,7 @@ import { useAnchorProgram } from '@providers/anchor';
 import { CacheEntry, FetchStatus } from '@providers/cache';
 import { useCluster } from '@providers/cluster';
 import { useTokenRegistry } from '@providers/mints/token-registry';
+import { PROGRAM_ID as ACCOUNT_COMPRESSION_ID } from '@solana/spl-account-compression';
```

Comment:
Did this end up pulling a crapton of code into this route, for just this one string? If so, maybe inline the string.

Reply:
I guess it doesn't matter, since this module is used elsewhere.